lib/datadog/profiling/exporter.rb

```diff
@@ -60,10 +59,9 @@ def flush
           start: start,
           finish: finish,
           pprof_file_name: Datadog::Profiling::Ext::Transport::HTTP::PPROF_DEFAULT_FILENAME,
-          pprof_data: Datadog::Core::Utils::Compression.gzip(uncompressed_pprof),
+          pprof_data: uncompressed_pprof.to_s,
```

Comment:
Is the data just left uncompressed here now? I assume it's going to be compressed later down the line, before it's flushed out of the process?

Reply:
Exactly; it goes from the `Profiling::Exporter` to the `Profiling::HttpTransport` uncompressed. The lifetime for the pprof data is quite short (until the report is successful), so I don't expect this to have a noticeable impact on application memory.